tests/Validation/ValidationValidatorTest.php

```diff
@@ -728,6 +728,33 @@ public function testPositionValuesAreReplaced()
         $this->assertTrue($v->passes());
     }
 
+    public function testArrayValidationMessagesAreAccurate()
+    {
+        // Note that the *second* email is invalid in the user input sample
+        $userInput = ['test@example.com', 'invalid_email', 'john@test.com'];
+
+        // Since the issue occurs between module boundaries (validation &
+        //  locale), we'll need to load the actual locale file.
+        $trans = $this->getIlluminateArrayTranslator();
+        $locale = require_once('src/Illuminate/Translation/lang/en/validation.php');
```

Comment:
require_once is the wrong function here, as running the test twice (with --times 2) will fail

Reply:
requiring the file at all is still pretty obscure tbh

Reply:
@GrahamCampbell Absolutely agreed, thank you for the great comments. Looking at the existing tests for validation, it doesn't appear that any actually validate the actual error messages. Do you have any suggestions on a better way to load the locale in the context of unit tests? Or should I just change `require_once` to just `require`?

Side note, this PR is for demonstration purposeses (for now at least) as requested on the original issue report.